hello-world/tutorial/index.md

```diff
@@ -6,128 +6,172 @@ order: 1
 
 The Hello World example is a time-honored tradition in computer programming.
 
-The Interblockchain Communication Protocol is an important part of the Cosmos SDK ecosystem. 
-Understanding how to create and send packets across blockchain will help you navigate between blockchains with the Cosmos SDK.
+The Inter-Blockchain Communication protocol (IBC) is an important part of the Cosmos SDK ecosystem.
+
+This tutorial builds an understanding of how to create and send packets across blockchain. This foundational knowledge helps you navigate between blockchains with the Cosmos SDK.
 
 **You will learn how to**
-- Use Interblockchain Communication Protocol (IBC) to create and send packets between blockchains.
+
+- Use IBC to create and send packets between blockchains.
 - Navigate between blockchains using the Cosmos SDK and the Starport Relayer.
 - Create a basic blog post and save the post on another blockchain.
 
-<i>Follow this tutorial on YouTube</i>
+_Follow this tutorial on YouTube_
 
-<a href="https://www.youtube.com/watch?v=NmytpuD33lY" target="_blank">
-<img src="https://i.ytimg.com/vi/NmytpuD33lY/hq720.jpg" alt="IBC Hello World Tutorial" width="150"/></a>
+[![IBC Hello World Tutorial](https://i.ytimg.com/vi/NmytpuD33lY/hq720.jpg)](https://www.youtube.com/watch?v=NmytpuD33lY)
 
 ## What is Cosmos SDK and IBC?
 
 The Cosmos SDK is a framework to create a blockchain app. The Cosmos SDK allows developers to easily create custom blockchains that can natively interoperate with other blockchains.
+
 The IBC module in the Cosmos SDK is the standard for the interaction between two blockchains. The IBC module defines how packets and messages are constructed to be interpreted by the sending and the receiving blockchain.
+
 The Cosmos IBC relayer package lets you can connect between sets of IBC-enabled chains. This tutorial teaches you how to create two blockchains and then start and use the relayer with Starport to connect two blockchains.
+
 This tutorial covers essentials like modules, IBC packets, relayer, and the lifecycle of packets routed through IBC.
 
-## Prerequisites 
+## Prerequisites
 
-This tutorial uses [Starport](https://github.com/tendermint/starport) v0.15.0. The Starport tool is the easiest way to build a blockchain. 
+This tutorial uses [Starport](https://github.com/tendermint/starport) v0.15.0\. The Starport tool is the easiest way to build a blockchain.
```

Comment:
What is the backslash for?

Reply:
oops that was a typo... in the future, maybe I can have a chance to address PR questions? Comments like this are useful, but merging without back and forth leaves us an opportunity to improve our process. Thanks for being quick, but maybe not so quick? :)